core/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -100,26 +98,6 @@
 public abstract class LoadBalancer {
   /**
    * Handles newly resolved server groups and metadata attributes from name resolution system.
-   * {@code servers} contained in {@link ResolvedServerInfoGroup} should be considered equivalent
-   * but may be flattened into a single list if needed.
-   *
-   * <p>Implementations should not modify the given {@code servers}.
-   *
-   * @deprecated Implement {@link #handleResolvedAddressGroups} instead.  As it is deprecated, the
-   *             {@link ResolvedServerInfo}s from the passed-in {@link ResolvedServerInfoGroup}s
-   *             lose all their attributes.
-   *
-   * @param servers the resolved server addresses, never empty.
-   * @param attributes extra metadata from naming system.
-   */
-  @Deprecated
-  public void handleResolvedAddresses(
-      List<ResolvedServerInfoGroup> servers, Attributes attributes) {
```

Comment:
ResolvedServerInfoGroup should also be removed.

Reply:
Done